code_search/code-search.ipynb

```diff
@@ -0,0 +1,451 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "-"
+    }
+   },
+   "source": [
+    "# Code Search on Kubeflow\n",
+    "\n",
+    "This notebook implements an end-to-end Semantic Code Search on top of [Kubeflow](https://www.kubeflow.org/) - given an input query string, get a list of code snippets semantically similar to the query string. \n",
+    "\n",
+    "\n",
+    "# TODO\n",
+    "\n",
+    "- Appropriate Working Directories for each of the bash commands\n",
+    "- `ks param set` requires a small refactor of the Ksonnet Application.\n",
+    "- Verify everything runs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "**NOTE**: If you are using this notebook on a Kubeflow cluster, all the pre-prequisites have already been set up.\n",
```

Comment:
Doesn't this depend on what notebook image they are using? Lets provide a link to the notebook image they should be  using.

Reply:
This is true. Thanks for pointing out.